app/assets/javascripts/components/course.cjsx

```diff
@@ -36,7 +36,7 @@ Course = React.createClass(
     @state.current_user
   submit: (e) ->
     e.preventDefault()
-    return unless confirm "Upon submission, this course will be automatically posted to Wikipedia with your user account. After that, new edits to the timeline will be mirrored to Wikipedia.\n\nAre you sure you want to do this?"
+    return unless confirm #{I18n.t("courses.warn_mirrored")}
```

Comment:
Here you only need the I18n.t() part. "#{}" is for string interpolation in coffeescript. The curly braces outside of a string, elsewhere in this file, are for adding coffeescript code within react cjsx fragments. This is just a function that returns a  string, so we need neither, I think.